### PR TITLE
use nixpkgs 20.03 by default

### PR DESCRIPTION
**Pull request by jbgi**
also removed skeleton.
jormungandr still built with 19.09 though.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -116,7 +116,7 @@ let
       (commonLib.pkgsDefault.callPackage ./overlays/rust/mozilla.nix {})
       (import ./overlays/rust)
     ];
-    pkgs = import sources.nixpkgs {
+    pkgs = import sources."nixpkgs-19.09" {
```
Comment (manveru) [inline default.nix:119]: When can we get rid of that?

Comment (manveru) [reply default.nix:119]: I guess when we bump Jormungandr :)

Comment (jbgi) [reply default.nix:119]: I think `nixpkgs-mozilla` rust overlay is not compatible with 20.03. Did not dig deep into the error though.

**Pull request merged**